docs/docs/sourcing-from-netlify-cms.md

```diff
@@ -144,6 +144,23 @@ backend:
 
 Now you can save the config.yml file, commit the change, and push it to your GitHub repo.
 
+#### Authenticating with GitLab
+
+See the [GitLab Backend](https://www.netlifycms.org/docs/authentication-backends/#gitlab-backend)
+section for details on how to configure authentication with GitLab.
+
+If you use the Client-Side Implicit Grant option, disable the Netlify Identity service in your
```

Comment:
```suggestion
If you use the [Client-Side Implicit Grant](https://www.netlifycms.org/docs/authentication-backends/#client-side-implicit-grant) option, disable the Netlify Identity service in your
```

How about also linking this? 😊

Reply:
Sure, good idea. Should I amend my commit or just commit your suggestion and let the merge/squash decision to the maintainer?

Reply:
You can click on "Commit suggestion" :)

![image](https://user-images.githubusercontent.com/16143594/55320270-fb452e80-5476-11e9-8c0b-35bf926f238e.png)

The PR gets squashed to one commit

Reply:
Thank you.